app/views_redesigned/shared/_index_layout.html.erb

```diff
@@ -0,0 +1,28 @@
+<%
+#  Example usage:
+#
+#  render "/shared/index_layout", locals: {
+#  title: "Images",
+#  buttons: [
+#    link_to("Upload Image", new_image_path, class: "btn btn-primary")
+#  ]
+%>
```

Comment:
probably should remove or use this more. I don't think it serves a good purpose being in here though. I would really love to do something like [this](https://github.com/github/actionview-component/issues/40#issuecomment-541221309) though..